pyproject.toml

```diff
@@ -49,7 +49,7 @@ pytest = ">=7,<9"
 pytest-cov = ">=3,<7"
 pytest-asyncio = ">=0.19,<0.26"
 pycairo = "^1.21.0"
-PyGObject = "^3.42.2"
+PyGObject = {version = ">=3.50,<3.51", python = ">=3.9,<4"}
```

Comment:
```suggestion
PyGObject = {version = ">=3.50,<3.51", python = "<4"}
```

Does this work so we don't have to bump the Python version in 2 places?

Reply:
probably.. testing now

Reply:
You are too fast. 😝

Reply:
https://github.com/Bluetooth-Devices/dbus-fast/compare/reduce_pin?expand=1

Reply:
https://github.com/Bluetooth-Devices/dbus-fast/pull/425